examples/c/examplePio.c

```diff
@@ -592,9 +598,11 @@ int main(int argc, char* argv[])
     pioExInst->cleanUp(pioExInst);
     
 #ifdef TIMING    
+#ifndef TIMING_INTERNAL
```

Comment:
Seems like you could have saved a lot of changes by leaving the TIMING macro to be defined for GPTL and using a different macro for your timer. THat way, you would not have to modify every use of #ifdef TIMING in this way. If you will make this change, many differences between your fork and the PIO master branch would be resolved, leaving the other differences much more visible.